client/verta/docs/tutorials/organizations/create_organization.rst

```diff
@@ -0,0 +1,32 @@
+Creating an organization
```

Comment:
@conradoverta I've been using title case for page and section headers, and I weakly prefer it, but do you have a particular preference for only capitalizing the first word?
```suggestion
Creating an Organization
```